.github/workflows/ci.yml

```diff
@@ -106,6 +107,7 @@ jobs:
           - rust: '1.56'
           - rust: nightly-2021-08-21 # Rust 1.56, LLVM 12
           - rust: '1.59' # LLVM 13
+          # XXX @brodycj TODO KEEP, REMOVE, OR ADD RATIONALE FOR COMMENTED-OUT ENTRIES - SEPARATE PR
```

Comment:
These are heavy to always be enabled, but they will be temporarily uncommented to see if they are affected by the LLVM bug if the code for this target is changed.

Reply:
raised PR #196 to propose comments for these commented entries